code/script.js

```diff
@@ -0,0 +1,107 @@
+const projects = document.getElementById('projects')
+const profile = document.getElementById('profile')
+const repos = document.getElementById('repos')
+
+const username = 'StephannieMedenilla'
+const USER_PROFILE = `https://api.github.com/users/SteppbySteph`
+const USER_REPO = `https://api.github.com/users/SteppbySteph/repos`
+
+
+
+const options = {
+    method: 'GET',
+    headers: {
+          Authorization: `token ${GITHUB_TOKEN}`
+    }
+}    
+
+
+//----USER NAME & PROFILE----//
+
+const getProfile = () => {
+    fetch(USER_PROFILE, options)
+        .then(res => res.json())
+        .then(data => {
+            const user = data.name
```

Comment:
I think actually that we should have our github user name included (SteppbySteph) - that will be fetched by user.login. It might be due user name is unique in github and therefor easier for anyone to find the correct Stephannie Medenilla. Although in your case there was only one Stephannie Medenilla when I did a search :-) I can see that some of our teammates have both name and user name.